src/Elasticsearch.Net.Aws/Elasticsearch.Net.Aws/Elasticsearch.Net.Aws.csproj

```diff
@@ -4,7 +4,7 @@
     <VersionPrefix>7.0.4</VersionPrefix>
     <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
     <AssemblyName>Elasticsearch.Net.Aws</AssemblyName>
-    <PackageId>Elasticsearch.Net.Aws</PackageId>
+    <PackageId>D2L.Elasticsearch.Net.Aws</PackageId>
```

Comment:
I don't want to change the package name.

Reply:
Sorry, this PR wasn't meant for you.  I opened it up against the wrong fork.